patches/@react-native-community+cameraroll+4.0.1.patch

```diff
@@ -1,5 +1,15 @@
+diff --git a/node_modules/@react-native-community/cameraroll/android/gradle.properties b/node_modules/@react-native-community/cameraroll/android/gradle.properties
+index 0bec1b5..a3dbe4e 100644
+--- a/node_modules/@react-native-community/cameraroll/android/gradle.properties
++++ b/node_modules/@react-native-community/cameraroll/android/gradle.properties
+@@ -1,4 +1,4 @@
+-ReactNativeCameraRoll_compileSdkVersion=28
++ReactNativeCameraRoll_compileSdkVersion=33
```

Comment:
this is already updated in the new versions of the library, can we consider upgrading it instead?

Reply:
We've decided to switch from `@react-native-community/cameraroll` to `@react-native-camera-roll/camera-roll`. I've created a new task for this migration.